### PR TITLE
updates and improvements

### PR DESCRIPTION
**Pull request by opZywl**
Nametags: improvise background and border option & better way to see Distance mode 

Updated SuperKnockBack

Added new modes, delay settings ground and onlymove options, now the modes tag is visible in the arraylist

updates in AntiBlind
now it is possible to configure the fire mode alpha and new bosshealth option (many servers get boss health that gets in the way of vision)

BlackStyle ClickGUI more sensitive colors and aver with its mode in options

EditorPanel with transparent view

### CODE REVIEW

**Comment by CzechHek** [comment]
Also **fixes** mc.netHandler.addToSendQueue(packet) being mistakenly replaced in PacketUtils along with all other mc.netHandler.addToSendQueue(packet) to just sendPacket(), **creating a loop of sendPacket calls when trying to send a packet**.

However, https://maven.minecraftforge.net/ is down and builds are failing.

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/SuperKnockback.kt [hunk 1]
```diff
@@ -7,32 +7,109 @@ package net.ccbluex.liquidbounce.features.module.modules.combat
...
-            mc.thePlayer.isSprinting = true
-            mc.thePlayer.serverSprintState = true
+                "sneakpacket" -> {
+                    if (mc.thePlayer.isSprinting) {
+                        mc.thePlayer.isSprinting = true
+                    }
+                    mc.netHandler.addToSendQueue(C0BPacketEntityAction(mc.thePlayer, C0BPacketEntityAction.Action.STOP_SPRINTING))
+                    mc.netHandler.addToSendQueue(C0BPacketEntityAction(mc.thePlayer, C0BPacketEntityAction.Action.START_SNEAKING))
+                    mc.netHandler.addToSendQueue(C0BPacketEntityAction(mc.thePlayer, C0BPacketEntityAction.Action.START_SPRINTING))
+                    mc.netHandler.addToSendQueue(C0BPacketEntityAction(mc.thePlayer, C0BPacketEntityAction.Action.STOP_SNEAKING))
```
Comment (1zun4secondary) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/SuperKnockback.kt]: If we have sendPacket, use it please. Thanks. Consistency.

Comment (1zun4secondary) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/SuperKnockback.kt]: Please resolve this so it can be merged.

Comment (opZywl) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/SuperKnockback.kt]: in few hours I back my house and send!! Thanks ~ sorry

Comment (1zun4secondary) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/SuperKnockback.kt]: already done

**Pull request merged**